sass-boilerplate

```diff
@@ -0,0 +1 @@
+Subproject commit b072e79e050e36178c00eb979a5bd466c9496c77
```

Comment:
NM i saw your message! `Git` ignores empty directories in your project. So a `hack` around it is to create an empty file. `Rails` does this by stuffing empty directories with a `.keep` file.

**TLDR** :
`touch app/assets/base/.keep` and then commit that.

SO question: http://stackoverflow.com/questions/17928581/random-concerns-folders-and-keep-files